spec/services/duplicate_passport_analytics_spec.rb

```diff
@@ -102,11 +102,11 @@
     end
 
     it "reports all reportable metrics on statsd" do
-      expect(Statsd.instance).to receive(:gauge).with("DuplicatePassportAnalytics.duplicate_passports_across_facilities.size", 2)
-      expect(Statsd.instance).to receive(:gauge).with("DuplicatePassportAnalytics.duplicate_passports_in_same_facility.size", 1)
-      expect(Statsd.instance).to receive(:gauge).with("DuplicatePassportAnalytics.duplicate_passports_across_districts.size", 0)
-      expect(Statsd.instance).to receive(:gauge).with("DuplicatePassportAnalytics.duplicate_passports_across_blocks.size", 1)
-      expect(Statsd.instance).to receive(:gauge).with("ReportDuplicatePassports.size", 2) # legacy
+      expect(Metrics.instance).to receive(:gauge).with("duplicate_passports_across_facilities", 2)
+      expect(Metrics.instance).to receive(:gauge).with("duplicate_passports_in_same_facility", 1)
+      expect(Metrics.instance).to receive(:gauge).with("duplicate_passports_across_districts", 0)
+      expect(Metrics.instance).to receive(:gauge).with("duplicate_passports_across_blocks", 1)
+      expect(Metrics.instance).to receive(:gauge).with("duplicate_passports_across_facilities", 2) # legacy
```

Comment:
**chore** (non-blocking): could mention in the comment why it's "legacy" and what the path to cleaning this up in the future could look like?

Reply:
I didn't add that line btw